More active voice language in quick start. (#600)
kubewarden/docs: docs/quick-start.md

Since
[`v0.4.0`](https://github.com/kubewarden/kubewarden-controller/releases/tag/v0.4.0),
a `PolicyServer` resource named `default` isn't created using the
`kubewarden-controller` chart. Now a Helm chart called `kubewarden-defaults`,
installs the default policy server.

This means that if not using the latest version of the `kubewarden-controller`
while upgrading or deleting, your default policy server isn't upgraded or
deleted. So, you might run into issues if you try to install the
`kubewarden-defaults` with conflicting information, for example, the same

> policy server name. To install future upgrades in the

Reviewer: ```suggestion
policy server name. To install future upgrades of the
```

| N | | - `registry`: policy download from an [OCI artifacts](https://github.com/opencontainers/artifacts) compliant container registry. Example: `registry://<OCI registry/policy URL>`. |
| N | | - `http`, `https`: policy download from a regular HTTP(s) server. Example: `https://<website/policy URL>`. |
| N | | - `file`: load the policy from a file in the computer file system. Example: `file:///<policy WASM binary full path>`. |
| Y | `resources` | The Kubernetes resources evaluated by the policy. |
| Y | `operations` | What operations for the previously given types to forward to this admission policy by the API server for evaluation. |
| Y | `mutating` | Set this boolean value to `true` for policies that can mutate incoming requests. |
| N | `settings` | A free-form object that contains the policy configuration values. |
| N | `failurePolicy` | The action to take if the request evaluated by a policy results in an error. Permitted options: |
| N | | - `Ignore`: ignore an error calling the webhook, and the API request continues. |
| N | | - `Fail`: an error calling the webhook causes the admission to fail and API request rejection. |

:::note

> You register the [[< cluster-admission-policy >]] resources with a `*`

Reviewer: ```suggestion
The controller registers the [[< cluster-admission-policy >]] resources with a `*`
```

clusteradmissionpolicy.policies.kubewarden.io/privileged-pods created
```

> When defining a [[< cluster-admission-policy >]], the status becomes

Reviewer: ```suggestion
After instantiating a [[< cluster-admission-policy >]], the status becomes
```
